src/camunda_worker/tasks/tests/test_task_create_status.py

```diff
@@ -0,0 +1,66 @@
+from django.test import TestCase
+
+import requests_mock
+from django_camunda.models import CamundaConfig
+from zgw_consumers.models import Service
+
+from camunda_worker.external_tasks.constants import Statuses
+from camunda_worker.external_tasks.models import FetchedTask
+
+from ..tasks import CreateStatusTask
+
+ZTC_URL = "https://openzaak.utrechtproeftuin.nl/catalogi/api/v1/"
+ZRC_URL = "https://openzaak.utrechtproeftuin.nl/zaken/api/v1/"
+STATUSTYPE = f"{ZTC_URL}statustypen/7ff0bd9d-571f-47d0-8205-77ae41c3fc0b"
+ZAAK = f"{ZRC_URL}zaken/4f8b4811-5d7e-4e9b-8201-b35f5101f891"
+STATUS = f"{ZRC_URL}statussen/b7218c76-7478-41e9-a088-54d2f914a713"
+
+
+@requests_mock.Mocker(real_http=True)
```

Comment:
Can you add a TODO to mock whatever requires `real_http`?

Reply:
TODO is added